packages/create-component/scaffolders/constants.js

```diff
@@ -0,0 +1,24 @@
+const renames = {
+    _gitignore: '.gitignore',
```

Comment:
why this renames needed?

Reply:
If the .gitignore file of the template is stored with the name .gitignore, the files will be ignored within the sub directory... So for safety purpose I have renames for, package.json, .gitignore, etc. So that we don't have to worry about special files.